lib/Parser/rterrors.h

```diff
@@ -20,6 +20,7 @@
 #define RT_ERROR_MSG_UNUSED_ENTRY
 #endif
 
+// NOTE: The VBSERR macros should not be changed due to legacy dependencies outside of Chakra.
```

Comment:
I don't know that legacy dependency is correct.  These are nice error message strings for general system HRESULTs which is certainly still applicable and not deprecated or defunct.